integrations/tensorflow/iree_tf_compiler/iree-import-tflite-main.cpp

```diff
@@ -91,15 +94,24 @@ int main(int argc, char **argv) {
                                         outputArrayFlag.end());
   auto loc = mlir::FileLineColLoc::get(&context,
                                        inputFile->getBufferIdentifier(), 0, 0);
-  OwningOpRef<mlir::ModuleOp> module = tflite::FlatBufferToMlir(
-      absl::string_view(inputFile->getBufferStart(),
-                        inputFile->getBufferSize()),
-      &context, loc,
-      /*use_external_constant=*/false, inputArrays, outputArrays);
-  if (!module) {
-    // Error should have emitted.
-    llvm::errs() << "Unable to import TFLite FlatBuffer to MLIR Module\n";
-    return 2;
+  OwningOpRef<mlir::ModuleOp> module;
+  if (loadMlirFile) {
```

Comment:
would be nice to make this automatic - checking that the header matches a tflite flatbuffer would let you tell (enough)

bytes 4 to 7 should be `TFL3`:
![image](https://user-images.githubusercontent.com/75337/180114767-86d52e0a-b361-41e1-bb46-0b9ca0a9f649.png)
`(string_view.substr(4, 4) == "TFL3")`

Reply:
Fixed. We may encounter some issues with files that don't have those bytes set (e.g. old TOCO files) but we can add an override if its an issue.